superset/datasets/commands/importers/v1/utils.py

```diff
@@ -127,9 +130,8 @@ def import_dataset(
     if dataset.id is None:
         session.flush()
 
-    example_database = get_example_database()
     try:
-        table_exists = example_database.has_table_by_name(dataset.table_name)
```

Comment:
This logic was wrong, but since it's only used when loading examples (where the dataset DB is the examples DB) it works fine.